Engine/DataFeeds/TimeSlice.cs

```diff
@@ -132,6 +146,15 @@ public static TimeSlice Create(DateTime utcDateTime, DateTimeZone algorithmTimeZ
             var futuresChains = new FuturesChains(algorithmTime);
             var symbolChanges = new SymbolChangedEvents(algorithmTime);
 
+            if (universeData.Count > 0)
+            {
+                // count universe data
+                foreach (var kvp in universeData)
+                {
+                    count += kvp.Value.Data.Count;
```

Comment:
idk everyone else, but I'm a big fan of counting **all** of the data points :grin: